components/filesystem/lock_handler.rst

```diff
@@ -62,8 +62,25 @@ You can pass an optional blocking argument as the first argument to the
 ``lock()`` method, which defaults to ``false``. If this is set to ``true``, your
 PHP code will wait indefinitely until the lock is released by another process.
 
-The resource is automatically released by PHP at the end of the script. In
-addition, you can invoke the
-:method:`Symfony\\Component\\Filesystem\\LockHandler::release` method to release
-the lock explicitly. Once it's released, any other process can lock the
-resource.
+.. caution::
+
+    Beware that the resource lock is automatically released as soon as PHP applies
```

Comment:
"Beware" should be used with "of". "Beware of the fact that the resource lock is automatically released"

Furthermore, I think "Be aware" is better in this context.